modules/api/.golangci.yml

```diff
@@ -77,6 +77,8 @@ issues:
   - cyclomatic complexity [0-9]+ of func `validateOpenStackCloudSpec` is high
   # gocritic
   - singleCaseSwitch # in most cases this is the beginning of a lookup table and should be kept an obvious table
+  # govet
+  - loop variable tc captured by func literal
```

Comment:
It's not exactly false positive, but I don't know how to deal with it. It complains about accessing outer tc while iterating over testcase in test run function. I don't know how we could refactor that. 
<img width="1739" alt="Screenshot 2023-04-07 at 09 30 41" src="https://user-images.githubusercontent.com/9121459/230564263-a9f4fe77-f520-4116-8eab-3d171522fdf0.png">